src/util/cilfacade.ml

```diff
@@ -8,10 +8,15 @@ module GU = Goblintutil
 include Cilfacade0
 
 (** Is character type (N1570 6.2.5.15)? *)
-let isCharType = function
+let isCharType = function (* TODO: also unrollType here? *)
```

Comment:
Probably. The only usage is in `cast_addr`. Might be worth trying what happens if we just do it.

Reply:
Did it here, no tests break at least. Not unrolling the type in such cases probably just makes us less precise at these checks if typedefs are used instead.